pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -98,8 +98,8 @@ func (alloc *Action) Execute(ssn *framework.Session) {
 	allNodes := ssn.NodeList
 	predicateFn := func(task *api.TaskInfo, node *api.NodeInfo) error {
 		// Check for Resource Predicate
-		if !task.InitResreq.LessEqual(node.FutureIdle(), api.Zero) {
-			return api.NewFitError(task, node, api.NodeResourceFitFailed)
+		if reason := task.InitResreq.LessEqualResource(node.FutureIdle(), api.Zero); len(reason) != 0 {
```

Comment:
Why add a new function here? If a return value is added on the basis of the original function, it indicates whether the resource information of the error is acceptable.

In addition, there are many places where lessEqual is called. Do other places need to be modified or modified iteratively?

Reply:
Why add a new function here?
Because the purpose of this PR is  to make volcano  event  belong to  a unscheduled pod ,  is more similar to kube-scheduler 。
So I do not need to change  many codes。
And the  LessEqual function is used by many places , if I change   LessEqual function , it is  a  huge project。
So I choose  a  better way to solve my problem ，and do not change volcano a  lot。

Reply:
If you are worried that the impact of the modification is too large, you can add a new function, but the return value of the new function is recommended to return two (bool, string), the first parameter indicates the result, and the second parameter indicates msg, in the return value character Whether the string is empty indicates the result of the judgment value, which feels inappropriate